api/repositories.go

```diff
@@ -105,6 +105,30 @@ func (r *Repositories) Delete(name, reason string, allowDataDeletion bool) error
 	return nil
 }
 
+type DefaultGroupEnum string
+
+func (r *Repositories) AddUser(name, username, group string) error {
```

Comment:
A user can have multiple default groups in a view. I think this method should accept a slice/varargs of groups to set.
Might be more appropriate to name it `AddOrUpdateUser` since in my experience it's add or update.

"Admin" includes all "Member" rights, but doesn't include "Eliminator". "Eliminator" only adds the two remaining permissions, but doesn't include any Member/Admin rights.